payment-service/src/main/java/com/lms/paymentservice/model/PaymentResponse.java

```diff
@@ -1,17 +1,39 @@
 package com.lms.paymentservice.model;
+/**
+ * This class encapsulates the response details of a payment transaction,
+ * including the success status, transaction ID, and an informational message.
+ *
+ * Shows the response to a payment request
+ *
+ * @author Callum Carroll
+ */
 
 public class PaymentResponse {
     private boolean success;
     private String transactionId;
     private String message;
 
+    /**
```

Comment:
I was going to let yourself do that so you could throw in what you authored to claim the work as yours but I can throw that in too if you'd like?

Reply:
I dont mind adding it to this PR, if you don't mind me modifying your PR, i think it would be easier than making a new PR for a one-line change. 
Alternatively you can add it to your pr, I don't mind